vite.config.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference types="vitest/config" />
+import { defineConfig } from "vite"
+
+export default defineConfig({
+	test: {
+		typecheck: {
+			tsconfig: "./tsconfig.json",
+		},
+		coverage: {
+			reporter: ["text", "json", "html"],
+			provider: "istanbul",
+		},
+		globals: true,
+		include: ["**/*.spec.[tj]s"],
+		testTimeout: 20000,
+		isolate: false,
+		exclude: ["node_modules", "dist"],
+		server: {
+			deps: {
+				inline: ["cryptly", "isly", "isoly", "typedly"],
```

Comment:
Should we put every repo here or should we put none and roll it out in dependency order?

Reply:
i have some sort of hope that with this new configuration this should no longer be needed. but i can fill it out with all of them for now.